util/mirror-tarballs

```diff
@@ -443,6 +443,13 @@ fi
 
 rm -f *.patch || exit 1
 
+answer=`$root/util/ver-ge "$main_ver" 1.17.8`
```

Comment:
We'll need to apply this patch to `ngx-build` script in `openresty-devel-utils` repo, BTW.